.github/workflows/hiera_tests.yaml

```diff
@@ -1,4 +1,4 @@
-name: Hiera Build
+name: Hiera Tests
```

Comment:
This change must also be reflected in the badge URL on line 5 in the README.md or that badge will loose track of its workflow.

Reply:
good catch, amended